packages/core/src/providers/yandex.ts

```diff
@@ -0,0 +1,58 @@
+import { OAuthConfig, OAuthUserConfig } from "."
+
+export interface YandexProfile {
+  id: string
+  login?: string
+  client_id: string
+  display_name: string
+  real_name: string
+  first_name: string
+  last_name?: string
+  sex?: string
+  default_email?: string
+  emails: string[]
+  psuid: string
+  /** bithday will be in YYYY-MM-DD formated string if exists */
+  birthday?: string
+  is_avatar_empty?: boolean
+  default_avatar_id?: string
+  default_phone?: { id: number; number: string }
+}
+
+export default function Yandex(
+  options: OAuthUserConfig<YandexProfile>
+): OAuthConfig<YandexProfile> {
+  return {
+    id: "yandex",
+    name: "Yandex",
+    type: "oauth",
+    // user will be prompted for rights which were set while creating OAuth app
+    authorization: "https://oauth.yandex.ru/authorize?scope=",
```

Comment:
To be compliant, we should pass those same scopes. What is the absolute minimum needed? Could you padd it here?